scripts/uninstall.sh

```
@@ -0,0 +1,68 @@
+    $SUDO systemctl stop llama-swap
+
+    status "Disabling llama-swap service..."
+    $SUDO systemctl disable llama-swap
+}
+if available systemctl; then
+    configure_systemd
+fi
+
+if available llama-swap; then
+    status "Removing llama-swap binary..."
+    $SUDO rm $(which llama-swap)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)